src/services/validations/validate.ts

```diff
@@ -54,6 +54,10 @@ const validateAddAuthSignerAddress = (address: string, selectedAcc: any): Valida
 
 const NOT_IN_ADDRESS_BOOK_MESSAGE =
   "This address isn't in your Address Book. Double-check the details before confirming."
+const FIRST_TIME_SEND_MESSAGE =
+  "You're trying to send to an address you've never sent funds to before (this history is stored only in your current browser). Double-check before confirming."
+const FIRST_TIME_SEND_IN_ADDRESS_BOOK_MESSAGE =
```

Comment:
These two should look more like a warning (or a note), not an actual error. They're too scary now.

<img width="636" height="400" alt="Screenshot 2025-09-15 at 15 20 32" src="https://github.com/user-attachments/assets/c4ace5a1-ec04-4000-b7f9-ae936f1d46c7" />